pyogrio/tests/test_geopandas_io.py

```diff
@@ -478,29 +523,67 @@ def test_write_dataframe_gdalparams(tmp_path, naturalearth_lowres):
     assert test_withindex_index_filename.exists() is True
 
 
-def test_write_dataframe_geometry_type(tmp_path, naturalearth_lowres):
-    df = read_dataframe(naturalearth_lowres)
+def test_write_dataframe_geometry_type_promote(tmp_path, naturalearth_lowres):
+    input_gdf = read_dataframe(naturalearth_lowres)
+    assert isinstance(input_gdf, gp.GeoDataFrame)
```

Comment:
I don't think it's needed to add this assert to all the tests

Reply:
pyLance started complaining about this somewhere in the past... but I just tested it, and it doesn't anymore, so I removed them.